pinot-controller/src/main/java/org/apache/pinot/controller/util/FileIngestionHelper.java

```diff
@@ -63,18 +69,13 @@
   private final TableConfig _tableConfig;
   private final Schema _schema;
   private final BatchConfig _batchConfig;
-  private final URI _controllerUri;
   private final File _uploadDir;
-  private final String _authToken;
 
-  public FileIngestionHelper(TableConfig tableConfig, Schema schema, BatchConfig batchConfig, URI controllerUri,
-      File uploadDir, String authToken) {
+  public FileIngestionHelper(TableConfig tableConfig, Schema schema, BatchConfig batchConfig, File uploadDir) {
```

Comment:
noooo. my auth token :(

it should be possible to provide auth info without writing it into the table config